src/sound_trigger/FingerprintSoundListener.py

```
@@ -0,0 +1,285 @@
+                    self._needs_reset = True
+                    continue
+
+                if self._needs_reset:
+                    self._reset_detectors()
+                    self._needs_reset = False
+
+                for _, detector in self._dodge_entries:
+                    detector.feed(chunk)
+                if self._counter_entry is not None:
+                    self._counter_entry[1].feed(chunk)
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)